troubleshoot-install.md

```diff
@@ -111,6 +111,22 @@ Alternatively, you can deploy Kubecost without persistent storage to store by fo
     --set prometheus.server.persistentVolume.enabled="false"
     ```
 
+## Issue: waiting for a volume to be created, either by external provisioner "ebs.csi.aws.com" or manually created by system administrator.
+
+If the PVC is in pending state for more than 5 minutes, the cluster is Amazon EKS 1.23+ the error message appears as the following example:
+
+```bash
+kubectl describe pvc cost-analyzer -n kubecost | grep "ebs.csi.aws.com"
+```
+Example result
+```
```

Comment:
Add blank line between code block and text.
Add language to code block

```suggestion

``` bash
```